indigo/docs/guides/howto-hello-indigo-tyrian.md

```diff
@@ -0,0 +1,418 @@
+---
+id: howto-responsive-ui-with-tyrian
+title: How to make a custom entity
+---
+
+## Responsive UI Using Tyrian
+
+There tend to be two primary types of UI in games: In-game UI (such as health over a
+player), and overlay (such as a HUD or dialog box). In-game UI tends not to
+worry too much about the screen size, and is dealt with primarily by the
+magnification configuration within Indigo. But what about overlays and HUD's?
+These tend to scale with the size of the screen and the position of such
+elements may change depending on the capabilities of the display. Many game
+engines provide a way to create overlay UI elements that scale in some way to
+the space they're in using a separate UI system. In Indigo we're exporting to
+JavaScript so it makes sense for us to use other web technologies for a problem
+that is already, ultimately, solved.
+
+Using Tyrian we can utilise HTML and CSS to ensure our overlay UI scales
+no matter what screen size we use.
+
+In this guide we'll take the basic `hello-indigo` example and add a button and
+a counter that scales with the size of the screen. It's a pretty basic example to
+be honest, but it should give you an idea of what can be achieved.
+
+## Setup
```

Comment:
There's a lot of fiddly detail in this write up - which is great, necessary - but it would be good to modify the narrative arc to make it smoother and easier to follow.

To do that, I'd love to see an "Overview" / "Here's the plan" / "This is what we're going to do" section before this setup block. Write out bullets of what needs to happen and I think in doing that exercise you may be able to reorder the things to avoid the reader jumping about too much.

Or maybe not, maybe it's just difficult! But worst case, at least the reader knows what's coming up.